content/cloud-docs/api-docs/state-version-outputs.mdx

```diff
@@ -153,3 +153,65 @@ curl \
   }
 }
 ```
+
+## Show Current State Version Outputs for a Workspace
+
+This endpoint allows users without permissions to read state versions to fetch outputs for a workspace.
+
+-> **Note:** Sensitive values are not revealed and will be returned as `null`.
```

Comment:
💯 I like that this is called out.

Reply:
Please add a note w/ link about how to fetch sensitive output values (you have to use the outputs/:id GET endpoint for a specific output, which is the the related -> self link